zuul-core/src/main/java/com/netflix/zuul/netty/server/Server.java

```diff
@@ -57,67 +71,94 @@
  */
 public class Server
 {
-    public static final DynamicBooleanProperty USE_EPOLL = new DynamicBooleanProperty("zuul.server.netty.socket.epoll", false);
+    /**
+     * This field is effectively a noop, as Epoll is enabled automatically if available.   This can be disabled by
+     * using the {@link #forceNio} property.
+     */
+    @Deprecated
+    public static final DynamicBooleanProperty USE_EPOLL =
+            new DynamicBooleanProperty("zuul.server.netty.socket.epoll", false);
+
+    /**
+     * If {@code true}, The Zuul server will avoid autodetecting the transport type and use the default Java NIO
+     * transport.
+     */
+    static final DynamicBooleanProperty forceNio =
```

Comment:
I went with lower case here because typically I have seen upper case used for constants. Since this is dynamic, I decided to lower it.    I think generally this pattern should apply, but I don't want to break compat, so I didn't update `USE_EPOLL`.

Reply:
Dynamic props are kind of like constants (they rarely change) so that's why you'll see them upper cased in a lot of places.

Reply:
Done, though this is still confusing IMO